docs/tutorials/tutorials-runtime-upgrade.md

```diff
@@ -49,6 +48,46 @@ sidebar_label: Runtime Upgrade
 
 * Click the _Submit proposal_ button and sign the transaction
 
+![Submit Proposal](/img/tutorial/submit_proposal.png "Submit Proposal")
+
+### Step 4: Seconding the Proposal
+
+* Select the account to second the proposal
+
+* Verify the number of tokens required to second the proposal and Click the "Second" button and sign the transaction
+
+![Second Proposal1](/img/tutorial/second_proposal1.png "Second Proposal1")
+
+![Second Proposal2](/img/tutorial/second_proposal2.png "Second Proposal2")
+
+![Second Proposal3](/img/tutorial/second_proposal3.png "Second Proposal3")
+
+### Step 5: Voting on the Proposal
+
+* The most seconded proposal moves to public referendum
```

Comment:
a nice to have would be to mention what happens to the proposal if nobody seconds the proposal.
does it automatically get moved to the public referendum for voting anyway?
and how many sessions or eras could go by before that happens, depending on the runtime configuration?
and to mention what happens if nobody votes on the referendum